cypress/e2e/tests/pages/explorer/apps/repositories.spec.ts

```diff
@@ -0,0 +1,53 @@
+import ReposListPagePo from '@/cypress/e2e/po/pages/repositories.po';
+import AppClusterRepoEditPo from '@/cypress/e2e/po/edit/catalog.cattle.io.clusterrepo.po';
+
+describe('Apps/Repositories', { tags: ['@explorer', '@adminUser'] }, () => {
+  it('Should reset input values when switching from Helm index URL to Git Repo and vice versa', () => {
+    cy.login();
```

Comment:
Thanks for the review and your suggestions too, I moved both `login` and `goTo` to the `before` block as it would run only once in this test unit. I didn't need to use `navTo` because the initial page for this test is already landed on the right nav item and only needs to click `Create` button on the same page. Please let me know if I missed something.